deploy/helm/kubecf/assets/operations/instance_groups/api.yaml

```diff
@@ -105,17 +105,17 @@
 - type: replace
   path: /variables/-
   value:
-    name: "ccdb_key_label_{{ $keyLabel }}"
+    name: "ccdb_key_label_{{ $keyLabel | lower }}"
     type: password
 {{- end }}
 - type: replace
   path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/cc/database_encryption?
   value: &encryption_info
     keys:
       {{- range $keyLabel := .Values.ccdb.encryption.rotation.key_labels }}
-      ccdb_key_label_{{ $keyLabel }}: "((ccdb_key_label_{{ $keyLabel }}))"
+      "{{ $keyLabel }}": "((ccdb_key_label_{{ $keyLabel | lower }}))"
```

Comment:
You don't need to quote anything before the `:`.
```suggestion
      {{ $keyLabel }}: "((ccdb_key_label_{{ $keyLabel | lower }}))"
```

Reply:
Ping @mook-as. This was my first suggestion when I first opened the PR to add these keys. Mark explicitly asked to add the prefix. Mark, any concerns about doing this now?

Reply:
> You don't need to quote anything before the `:`

@f0rmiga I know, but then you don't need to quote the right hand side either. I suspect all the quotes are unnecessary, but if we quote `$keylabel` on the right side, then we should quote it on the left for consistency as well.

Reply:
I asked for the prefix in https://github.com/cloudfoundry-incubator/kubecf/pull/298#discussion_r365026734 because it was just a top level variable: when looking at an arbitrary kubecf deployment, seeing a secret named `var-foo` would be rather confusing (at least, more so than `var-ccdb-key-label-foo`, IMHO).  But that only applied to the _variable name_; it doesn't apply to the hash key here. So this change should be fine.

Reply:
@jandubois Not really, on the right side, the quote is intended to make the data a string. On the left side it's always a string.

Reply:
@f0rmiga You will be surprised by this then:

```
$ ruby -ryaml -e 'puts YAML.load("not: true")'
{"not"=>true}

$ ruby -ryaml -e 'puts YAML.load("true: true")'
{true=>true}

$ ruby -ryaml -e 'puts YAML.load("\"true\": true")'
{"true"=>true}
```

So without the quotes on the left side, the value is not guaranteed to be a string either...

Reply:
Or this:

```
$ ruby -ryaml -e 'puts YAML.load("0x7a: true")'
{122=>true}
```

Reply:
Tempted to put all the quotes back in now 😄

Reply:
![image](https://media.giphy.com/media/xT0xeJpnrWC4XWblEk/giphy-downsized.gif)